curriculum/challenges/english/05-apis-and-microservices/basic-node-and-express/meet-the-node-console.english.md

```diff
@@ -12,7 +12,11 @@ We recommend to keep the log panel open while working at these challenges. By re
 
 ## Instructions
 <section id='instructions'>
-Modify the <code>myApp.js</code> file to log "Hello World" to the console. 
+  
+  * Start a new project on Glitch using [this link](https://glitch.com/#!/import/github/freeCodeCamp/boilerplate-express/)  
```

Comment:
```suggestion
* If you have not already done so, please read the instructions in [the introduction](/learn/apis-and-microservices/basic-node-and-express/) and start a new project on Glitch using [this link](https://glitch.com/#!/import/github/freeCodeCamp/boilerplate-express/).
```
I think it's worth directing people to the original source of the info, in case they skimmed over it.

Reply:
That makes sense, thanks for pointing that out. Love the suggestion, and putting the link here as well.